workbase/BUILD.bazel

```diff
@@ -0,0 +1,94 @@
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+exports_files([
+    "package.json",
+    "package-lock.json",
+])
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "node_modules",
+    srcs = glob(
+        include = ["node_modules/**/*", "node_modules/.bin/*"],
+        exclude = [
+          # Files under test & docs may contain file names that
+          # are not legal Bazel labels (e.g.,
+          # node_modules/ecstatic/test/public/中文/檔案.html)
+          "node_modules/test/**",
+          "node_modules/docs/**",
+          # Files with spaces are not allowed in Bazel runfiles
+          # See https://github.com/bazelbuild/bazel/issues/4327
+          "node_modules/**/* */**",
+          "node_modules/**/* *",
+        ],
+    ),
+)
+
+
+genrule(
+    name = "distribution",
+    cmd = "$(location //workbase/wrappers-bazel:npm_run_build.sh) $(location workbase.zip)",
+    srcs = glob([
+        "src/**/*",
+        "build/**/*",
+        "static/**/*",
+        ".electron-vue/**/*",
+        "package.json",
+        ".babelrc",
+        "config.js",
+        "jsconfig.json"]) + [":node_modules"],
+    outs = ["workbase.zip"],
```

Comment:
what's the name of the current distribution zip? can we name it `grakn-core-workbase`? When we deploy, it should also include `VERSION`. Basically it should follow the same naming convention as our jars.

Reply:
@haikalpribadi currently it is `grakn-workbase-{VERSION}-{platform}.zip`
We'll be able to rename it while storing artifact on CircleCI or when uploading with `ghr` but not in `bazel` -- `genrule`'s `outs` is `nonconfigurable` which basically means "has to have static name"

Reply:
Okay, I see. Let's name it `grakn-workbase.zip` at least then, @vmax